src/server/api/data.ts

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import "../config";
+
+interface Coordinates {
+  latitude: number;
+  longitude: number;
+}
+
+const API_KEY = process.env.TICKET_MASTER_API_KEY;
+
+export const getCoordinates = async (postcode: any): Promise<Coordinates> => {
+  const response = await axios.get(
+    `https://api.postcodes.io/postcodes/${postcode}`
+  );
+
+  const lat: number = response.data.result.latitude;
+  const long: number = response.data.result.longitude;
+
+  return {
+    latitude: lat,
+    longitude: long,
+  };
+};
+
+export const getNearestConcert = async (
+  radius: number,
+  unit: string,
+  geohash: any
+): Promise<{}> => {
```

Comment:
If this function was part of a common library, as a consumer, I wouldn't know what `getNearestConcert` return, `{}` is not accurate enough. I would define a `TicketmasterReponse` type to better type this.

Reply:
updated code - https://github.com/builtbybea/bass-and-treble/pull/2/commits/7f5ddd2e7fc027b1d555901210b718fc7ff00a9a